java-processors/src/main/java/com/dataiku/dip/plugins/tradearea/PolygonBuilder.java

```diff
@@ -0,0 +1,26 @@
+package com.dataiku.dip.plugins.tradearea;
+
+import com.dataiku.dip.shaker.types.GeoPoint;
+
+public class PolygonBuilder {
+
+    StringBuilder wktPolygon;
```

Comment:
add final (the variable won't be reassigned after the constructor has been called)
```suggestion
    final StringBuilder wktPolygon;
```

Reply:
Done in 4a72e51